### PR TITLE
Add more verbs

### PR DESCRIPTION
**Pull request by norwd**
I'm using this list of verbs to validate PR titles, I've found that some verbs are missing (particularly development related terms like "update", "merge",  "encode" etc...)

I hope these additions are helpful to someone out there 🙂

### CODE REVIEW

File: data/words/verbs.json [hunk 1]
```diff
@@ -245,10 +289,18 @@
             "present": "branch",
             "past": "branched"
         },
+        {
+            "present": "break",
+            "past": "breaked"
```
Comment (hugovk) [inline data/words/verbs.json:294]:
```suggestion
            "past": "broke"
```

File: data/words/verbs.json [hunk 2]
```diff
@@ -677,6 +813,14 @@
             "present": "empty",
             "past": "emptied"
         },
+        {
+            "present": "enable",
+            "past": "endabled"
```
Comment (hugovk) [inline data/words/verbs.json:818]:
```suggestion
            "past": "enabled"
```

File: data/words/verbs.json [hunk 3]
```diff
@@ -1769,18 +2057,74 @@
...
+            "present": "resign",
+            "past": "resigned"
+        },
+        {
+            "present": "resolve",
+            "past": "resolved"
+        },
+        {
+            "present": "restore",
+            "past": "restore"
```
Comment (hugovk) [inline data/words/verbs.json:2082]:
```suggestion
            "past": "restored"
```

**Review by hugovk — CHANGES_REQUESTED** [review]
To fix / become fixed:

**Comment by norwd** [comment]
> To fix / become fixed:

Fixed now, sorry my spelling is awful 😅

**Review by hugovk — APPROVED** [review]
Thanks! I wonder why the CI is taking so long. Let's give it a bit more time.

**Comment by hugovk** [comment]
Please could you merge or rebase on `master`?

The old `ubuntu-18.04` CI runner is longer available, https://github.com/dariusk/corpora/pull/421 fixes it.

**Comment by hugovk** [comment]
Merge / merged, thanks!

**Comment by norwd** [comment]
> Please could you merge or rebase on `master`?

Ah sorry it was night time for me, looks like it worked out eventually. Thanks! 😄

**Comment by hugovk** [comment]
You did exactly what I asked for, so thank you!

**Pull request merged**